tests/integration.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
   globals: {
     puppeteerArguments: ['--no-sandbox', '--disable-setuid-sandbox', '--disable-dev-shm-usage'],
     APP_URL: 'http://localhost:3000/maps',
+    __config: require('@qwant/nconf-builder').get_without_check(),
```

Comment:
Is that not a bit confusing to define a configuration object here, different from the overridden configuration defined in /tests/integration/server_start.js ?

Reply:
Ah yes, maybe, I wasn't sure about that…
The thing is, because of changes in how modules are imported, integration tests are now broken if there is no `__config` global variable (it breaks [here](https://github.com/QwantResearch/erdapfel/blob/34c34eb14cb29aa92c96f38a9d29acc8f6f64fb0/local_modules/nconf_getter/index.js#L2), when `nconf_getter` is imported by `idunn_poi`).
Maybe there is a way in `server_start` to inject the resulting config object as a global variable for the client?